examples/private-data/Rocket.toml

```diff
@@ -0,0 +1,2 @@
+[default]
+secret_key = "itlYmFR2vYKrOmFhupMIn/hyB6lYCCTXz4yaQX89XVg="
```

Comment:
We should avoid committing any secret key to GitHub, even for examples. Rocket should auto-generate a random key on every launch, so we should be okay with not specifying one here.

Reply:
Understood, then I'll just delete Rocket.toml for this example, and Rocket will generate a new secret key with every run. 
I also deleted the secret key from the [cookies example](https://github.com/rwf2/Rocket/blob/c303add07fe15520654aa388e6f387979704b1d5/examples/cookies/Rocket.toml#L2).